### PR TITLE
fix: LW-10549 add missing onboarding and multiwallet posthog events, improve type-safety

### PR DESCRIPTION
**Pull request by szymonmaslowski**
# Checklist

- [x] JIRA - [\<link>](https://input-output.atlassian.net/browse/LW-10549)
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

Extracted onboarding and multi-wallet posthog events from the `PosthogAction` enum to be separate entities. Improved type safety around those events.

## Testing

Onboarding and multi-wallet flows send identical posthog events (same number of events, same time of sending) - same event texts except for the prefix which is either `onboarding` or `multiwalet`.

### CODE REVIEW

**Review by shawnbusuttil — COMMENTED** [review]
Left just one comment.

File: apps/browser-extension-wallet/src/views/browser-view/features/wallet-setup/types.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import { PostHogAction, PostHogProperties } from '@providers/AnalyticsProvider/analyticsTracker';
+import { PostHogProperties } from '@providers/AnalyticsProvider/analyticsTracker';
 
 export type SendOnboardingAnalyticsEvent = (
-  postHogAction: PostHogAction,
+  postHogAction: string,
```
[the remark this answers is not in the document]
Comment (szymonmaslowski) [reply apps/browser-extension-wallet/src/views/browser-view/features/wallet-setup/types.ts:4]: This file is dead. It is old onboarding and I have it removed on a different branch.

[the remark this answers is not in the document]
Comment (szymonmaslowski) [reply apps/browser-extension-wallet/src/views/browser-view/features/wallet-setup/types.ts:4]: More context: as you can see this PR is based on the codebase unification PR where on ording implementation was swapped with the adjusted multi wallet implementation and all the old code resides there without being used including this file.

**Review by shawnbusuttil — APPROVED** [review]

**Review by greatertomi — APPROVED** [review]
Looks good to me.

**Review by oldGreg5 — APPROVED** [review]

**Pull request merged**